care/facility/tasks/patient/discharge_report.py

```diff
@@ -0,0 +1,58 @@
+import datetime
+import random
+import string
+import time
+
+import celery
```

Comment:
Codacy found an issue: [Unable to import 'celery'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5811718)